taskcluster/kinds/analyze-corpus/kind.yml

```diff
@@ -0,0 +1,83 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - translations_taskgraph.transforms.from_datasets:per_dataset
+    - taskgraph.transforms.task_context
+    - taskgraph.transforms.run:transforms
+    - translations_taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - dataset
+
+tasks:
+    "{provider}-{dataset_sanitized}-{src_locale}-{trg_locale}":
+        description: Analyze {provider} {dataset_sanitized} dataset {src_locale}-{trg_locale}
+        attributes:
+            dataset-category: train
+            stage: analyze-corpus
+            cache:
+                type: analyze-corpus
+                resources:
+                    - pipeline/data/analyze.py
+                    - pipeline/data/requirements/analyze.txt
+        task-context:
+            from-parameters: {}
+            substitution-fields:
+                - description
+        worker-type: b-cpu-largedisk
+        dataset-config:
+            category: train
+            substitution-fields:
+                - description
+                - name
+                - dependencies
+                - fetches
+                - worker.env
+                - run.command
+        worker:
+            docker-image: {"in-tree": "train"}
+            max-run-time: 86400 # one day
+            artifacts:
+                - name: public/build
+                  path: /builds/worker/artifacts
+                  type: directory
+            env: {}
+
+        # Don't run unless explicitly scheduled
+        run-on-tasks-for: []
+    
+        run:
+            using: run-task
+            command:
+                - bash
+                - -c
+                - >-
+                    pip3 install --upgrade pip setuptools &&
+                    pip3 install -r $VCS_PATH/pipeline/data/requirements/analyze.txt
+                    &&
+                    python3 $VCS_PATH/pipeline/data/analyze.py
```

Comment:
Maybe it works for now but I assume we'll want to do some analysis based on parallel sentences, like the ratio between the number of words etc. We might want to tune filters based on those per language. In this case, the analyze script would need to accept either two files as input.